<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
    <class>MainWindow</class>
    <widget class="QMainWindow" name="MainWindow">
        <property name="geometry">
            <rect>
                <x>0</x>
                <y>0</y>
                <width>428</width>
                <height>926</height>
            </rect>
        </property>
        <property name="minimumSize">
            <size>
                <width>428</width>
                <height>600</height>
            </size>
        </property>
        <property name="maximumSize">
            <size>
                <width>428</width>
                <height>926</height>
            </size>
        </property>
        <property name="windowTitle">
            <string>Seminary Management - Login</string>
        </property>
        <property name="styleSheet">
            <string notr="true">
                QMainWindow {
                    background-color: #FFFFFF !important;
                    border-radius: 50px;
                    color: #000000 !important;
                }
                QWidget {
                    background-color: #FFFFFF !important;
                    color: #000000 !important;
                }
                * {
                    font-family: 'Poppins', sans-serif;
                }
            </string>
        </property>
        <widget class="QWidget" name="centralwidget">
            <property name="autoFillBackground">
                <bool>false</bool>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout">
                <item>
                    <spacer name="horizontalSpacer_left">
                        <property name="orientation">
                            <enum>Qt::Orientation::Horizontal</enum>
                        </property>
                        <property name="sizeHint" stdset="0">
                            <size>
                                <width>40</width>
                                <height>20</height>
                            </size>
                        </property>
                    </spacer>
                </item>
                <item>
                    <layout class="QVBoxLayout" name="verticalLayout_main">
                        <item>
                            <spacer name="verticalSpacer_top">
                                <property name="orientation">
                                    <enum>Qt::Orientation::Vertical</enum>
                                </property>
                                <property name="sizeHint" stdset="0">
                                    <size>
                                        <width>20</width>
                                        <height>20</height>
                                    </size>
                                </property>
                            </spacer>
                        </item>
                        <item>
                            <widget class="QWidget" name="contentWidget" native="true">
                                <property name="minimumSize">
                                    <size>
                                        <width>428</width>
                                        <height>600</height>
                                    </size>
                                </property>
                                <property name="maximumSize">
                                    <size>
                                        <width>428</width>
                                        <height>600</height>
                                    </size>
                                </property>
                                <widget class="QWidget" name="headerWidget" native="true">
                                    <property name="geometry">
                                        <rect>
                                            <x>30</x>
                                            <y>20</y>
                                            <width>361</width>
                                            <height>151</height>
                                        </rect>
                                    </property>
                                    <widget class="QLabel" name="logoLabel">
                                        <property name="geometry">
                                            <rect>
                                                <x>115</x>
                                                <y>10</y>
                                                <width>130</width>
                                                <height>130</height>
                                            </rect>
                                        </property>
                                        <property name="styleSheet">
                                            <string notr="true">
                                                QLabel {
                                                    border-radius: 65px;
                                                    background-color: #FFFFFF !important;
                                                    border: 3px solid #1F41BB;
                                                    padding: 5px;
                                                    qproperty-alignment: AlignCenter;
                                                }
                                            </string>
                                        </property>
                                        <property name="text">
                                            <string />
                                        </property>
                                        <property name="pixmap">
                                            <pixmap>
                                                c:/Users/Nguye/Documents/seminary-management/images/logo.png
                                            </pixmap>
                                        </property>
                                        <property name="scaledContents">
                                            <bool>true</bool>
                                        </property>
                                        <property name="alignment">
                                            <set>Qt::AlignmentFlag::AlignCenter</set>
                                        </property>
                                    </widget>
                                </widget>
                                <widget class="QWidget" name="formLayoutWidget">
                                    <property name="geometry">
                                        <rect>
                                            <x>31</x>
                                            <y>230</y>
                                            <width>366</width>
                                            <height>369</height>
                                        </rect>
                                    </property>
                                    <layout class="QFormLayout" name="formLayout">
                                        <property name="verticalSpacing">
                                            <number>20</number>
                                        </property>
                                        <item row="0" column="0">
                                            <widget class="QGroupBox" name="loginGroup">
                                                <property name="styleSheet">
                                                    <string notr="true">
                                                        QGroupBox {
                                                            border: none;
                                                            margin: 0px;
                                                            padding: 0px;
                                                        }
                                                    </string>
                                                </property>
                                                <property name="title">
                                                    <string />
                                                </property>
                                                <layout class="QVBoxLayout" name="verticalLayout">
                                                    <property name="spacing">
                                                        <number>29</number>
                                                    </property>
                                                    <property name="leftMargin">
                                                        <number>0</number>
                                                    </property>
                                                    <property name="topMargin">
                                                        <number>0</number>
                                                    </property>
                                                    <property name="rightMargin">
                                                        <number>0</number>
                                                    </property>
                                                    <property name="bottomMargin">
                                                        <number>0</number>
                                                    </property>
                                                    <item>
                                                        <widget class="QLineEdit" name="emailInput">
                                                            <property name="minimumSize">
                                                                <size>
                                                                    <width>357</width>
                                                                    <height>52</height>
                                                                </size>
                                                            </property>
                                                            <property name="maximumSize">
                                                                <size>
                                                                    <width>357</width>
                                                                    <height>52</height>
                                                                </size>
                                                            </property>
                                                            <property name="styleSheet">
                                                                <string notr="true">
                                                                    QLineEdit {
                                                                        width: 357px;
                                                                        height: 52px;
                                                                        padding: 12px 21px 12px 12px;
                                                                        background-color: #F1F4FF !important;
                                                                        border: 2px solid #1F41BB;
                                                                        border-radius: 8px;
                                                                        font-family: 'Poppins';
                                                                        font-size: 16px;
                                                                        font-weight: 500;
                                                                        color: #000000 !important;
                                                                    }
                                                                    QLineEdit::placeholder {
                                                                        color: #626262 !important;
                                                                        font-weight: 500;
                                                                    }
                                                                </string>
                                                            </property>
                                                            <property name="placeholderText">
                                                                <string>Email</string>
                                                            </property>
                                                        </widget>
                                                    </item>
                                                    <item>
                                                        <widget class="QLineEdit"
                                                            name="passwordInput">
                                                            <property name="minimumSize">
                                                                <size>
                                                                    <width>357</width>
                                                                    <height>52</height>
                                                                </size>
                                                            </property>
                                                            <property name="maximumSize">
                                                                <size>
                                                                    <width>357</width>
                                                                    <height>52</height>
                                                                </size>
                                                            </property>
                                                            <property name="styleSheet">
                                                                <string notr="true">
                                                                    QLineEdit {
                                                                        width: 357px;
                                                                        height: 52px;
                                                                        padding: 12px 21px 12px 12px;
                                                                        background-color: #F1F4FF !important;
                                                                        border: 2px solid #1F41BB;
                                                                        border-radius: 8px;
                                                                        font-family: 'Poppins';
                                                                        font-size: 16px;
                                                                        font-weight: 500;
                                                                        color: #000000 !important;
                                                                    }
                                                                    QLineEdit::placeholder {
                                                                        color: #626262 !important;
                                                                        font-weight: 500;
                                                                    }
                                                                </string>
                                                            </property>
                                                            <property name="echoMode">
                                                                <enum>QLineEdit::EchoMode::Password</enum>
                                                            </property>
                                                            <property name="placeholderText">
                                                                <string>Password</string>
                                                            </property>
                                                        </widget>
                                                    </item>
                                                </layout>
                                            </widget>
                                        </item>
                                        <item row="1" column="0">
                                            <widget class="QLabel" name="forgotPasswordLink">
                                                <property name="minimumSize">
                                                    <size>
                                                        <width>357</width>
                                                        <height>21</height>
                                                    </size>
                                                </property>
                                                <property name="maximumSize">
                                                    <size>
                                                        <width>357</width>
                                                        <height>21</height>
                                                    </size>
                                                </property>
                                                <property name="styleSheet">
                                                    <string notr="true">
                                                        QLabel {
                                                            font-family: 'Poppins';
                                                            font-weight: 600;
                                                            font-size: 14px;
                                                            line-height: 21px;
                                                            color: #1F41BB !important;
                                                            border: none;
                                                            background: transparent;
                                                            qproperty-alignment: 'AlignRight |
                                                            AlignVCenter';
                                                            text-align: right;
                                                            margin-right: 0px;
                                                            padding-right: 0px;
                                                        }
                                                        QLabel:hover {
                                                            text-decoration: underline;
                                                        }
                                                    </string>
                                                </property>
                                                <property name="text">
                                                    <string>Forgot your password?</string>
                                                </property>
                                                <property name="alignment">
                                                    <set>Qt::AlignRight|Qt::AlignVCenter</set>
                                                </property>
                                            </widget>
                                        </item>
                                        <item row="2" column="0">
                                            <widget class="QGroupBox" name="actionsGroup">
                                                <property name="minimumSize">
                                                    <size>
                                                        <width>357</width>
                                                        <height>131</height>
                                                    </size>
                                                </property>
                                                <property name="maximumSize">
                                                    <size>
                                                        <width>357</width>
                                                        <height>131</height>
                                                    </size>
                                                </property>
                                                <property name="styleSheet">
                                                    <string notr="true">
                                                        QGroupBox {
                                                            border: none;
                                                            margin: 0px;
                                                            padding: 0px;
                                                        }
                                                    </string>
                                                </property>
                                                <property name="title">
                                                    <string />
                                                </property>
                                                <layout class="QVBoxLayout" name="actionsLayout">
                                                    <property name="spacing">
                                                        <number>30</number>
                                                    </property>
                                                    <property name="leftMargin">
                                                        <number>0</number>
                                                    </property>
                                                    <property name="topMargin">
                                                        <number>0</number>
                                                    </property>
                                                    <property name="rightMargin">
                                                        <number>0</number>
                                                    </property>
                                                    <property name="bottomMargin">
                                                        <number>0</number>
                                                    </property>
                                                    <item>
                                                        <widget class="QPushButton"
                                                            name="submitButton">
                                                            <property name="minimumSize">
                                                                <size>
                                                                    <width>357</width>
                                                                    <height>52</height>
                                                                </size>
                                                            </property>
                                                            <property name="maximumSize">
                                                                <size>
                                                                    <width>357</width>
                                                                    <height>52</height>
                                                                </size>
                                                            </property>
                                                            <property name="styleSheet">
                                                                <string notr="true">
                                                                    QPushButton {
                                                                        width: 357px;
                                                                        height: 52px;
                                                                        background-color: #1F41BB
                                                                        !important;
                                                                        border-radius: 8px;
                                                                        border: none;
                                                                        font-family: 'Poppins';
                                                                        font-weight: 600;
                                                                        font-size: 17px;
                                                                        color: #FFFFFF !important;
                                                                        text-align: center;
                                                                    }
                                                                    QPushButton:hover {
                                                                        background-color: #1a3aa3 !important;
                                                                    }
                                                                    QPushButton:pressed {
                                                                        background-color: #16328a !important;
                                                                    }
                                                                </string>
                                                            </property>
                                                            <property name="text">
                                                                <string>Sign in</string>
                                                            </property>
                                                        </widget>
                                                    </item>
                                                </layout>
                                            </widget>
                                        </item>
                                    </layout>
                                </widget>
                            </widget>
                        </item>
                        <item>
                            <spacer name="verticalSpacer_bottom">
                                <property name="orientation">
                                    <enum>Qt::Orientation::Vertical</enum>
                                </property>
                                <property name="sizeHint" stdset="0">
                                    <size>
                                        <width>20</width>
                                        <height>80</height>
                                    </size>
                                </property>
                            </spacer>
                        </item>
                    </layout>
                </item>
                <item>
                    <spacer name="horizontalSpacer_right">
                        <property name="orientation">
                            <enum>Qt::Orientation::Horizontal</enum>
                        </property>
                        <property name="sizeHint" stdset="0">
                            <size>
                                <width>40</width>
                                <height>20</height>
                            </size>
                        </property>
                    </spacer>
                </item>
            </layout>
        </widget>
        <widget class="QStatusBar" name="statusbar" />
    </widget>
    <resources />
    <connections />
</ui>